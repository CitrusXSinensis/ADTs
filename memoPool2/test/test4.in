ra1 2 4 8 16 32 64 128 256 512x
pa
ca
aa 1024
ca
fa 511
pa

