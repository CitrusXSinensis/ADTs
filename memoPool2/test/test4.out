1 2 4 8 16 32 64 128 256 512
10/16
11/16
Chunk: 512
1 2 4 8 16 32 64 128 256 1024
